File: backend/integrations/paddle.py
from .env import get_env


def get_paddle_config() -> dict:
    return {
        "api_key": get_env("PADDLE_API_KEY", ""),
        "environment": get_env("PADDLE_ENV", "sandbox"),
        "price_id": get_env("PADDLE_PRICE_ID", ""),
    }

File: src/integrations/paddle.js
export function getPaddleConfig() {
  return {
    environment: import.meta.env.VITE_PADDLE_ENV || "sandbox",
    clientToken: import.meta.env.VITE_PADDLE_CLIENT_TOKEN || "",
    priceId: import.meta.env.VITE_PADDLE_PRICE_ID || "",
  };
}

export function isPaddleConfigured() {
  const config = getPaddleConfig();
  return Boolean(config.clientToken && config.priceId);
}
